python/instrumentation/openinference-instrumentation-dspy/examples/lm.py

```diff
@@ -0,0 +1,19 @@
+from dspy import LM
+from opentelemetry import trace as trace_api
+from opentelemetry.exporter.otlp.proto.http.trace_exporter import OTLPSpanExporter
+from opentelemetry.sdk import trace as trace_sdk
+from opentelemetry.sdk.trace.export import ConsoleSpanExporter, SimpleSpanProcessor
+
+from openinference.instrumentation.dspy import DSPyInstrumentor
+
+endpoint = "http://localhost:6006/v1/traces"
+tracer_provider = trace_sdk.TracerProvider()
+tracer_provider.add_span_processor(SimpleSpanProcessor(OTLPSpanExporter(endpoint)))
+tracer_provider.add_span_processor(SimpleSpanProcessor(ConsoleSpanExporter()))
+trace_api.set_tracer_provider(tracer_provider=tracer_provider)
+DSPyInstrumentor().instrument(tracer_provider=tracer_provider)
+
+
+lm = LM("openai/gpt-4")
```

Comment:
```suggestion
lm = LM("openai/gpt-4o-mini")
```

Reply:
Good idea. Will address in a later PR so I don't have to rebase.